Jellyfin.Plugin.Tvdb/ScheduledTasks/UpdateTask.cs

```diff
@@ -72,7 +72,8 @@ public async Task ExecuteAsync(IProgress<double> progress, CancellationToken can
             progress.Report(10);
             MetadataRefreshOptions refreshOptions = new MetadataRefreshOptions(new DirectoryService(_fileSystem))
             {
-                MetadataRefreshMode = MetadataRefreshMode.FullRefresh
+                MetadataRefreshMode = MetadataRefreshMode.FullRefresh,
```

Comment:
This will have some impact on manually edited data, but since the scheduled task is disabled by default (at least I think it is or should) it should be fine.
Do you think it would be beneficial to make this configurable?

Reply:
The goal is to keep the data up to date with the metadata provider so it has to overwrite existing data. Don't think should be configurable.

Reply:
Data won't be overwritten if the field is unchecked tho